torchvision/prototype/features/_bounding_box.py

```diff
@@ -140,23 +131,10 @@ def rotate(
         fill: FillTypeJIT = None,
         center: Optional[List[float]] = None,
     ) -> BoundingBox:
-        output = self._F.rotate_bounding_box(
+        output, image_size = self._F.rotate_bounding_box(
             self, format=self.format, image_size=self.image_size, angle=angle, expand=expand, center=center
         )
-        image_size = self.image_size
-        if expand:
-            # The way we recompute image_size is not optimal due to redundant computations of
```

Comment:
And an argument for adopting this, is the potential performance gains (and the obvious improvement in code-quality) from avoid recomputing staff. It would be great to get a benchmark on this to see what kind of difference we see. Previously we noted that bbox operations are a bottleneck.